cartoframes/data/clients/bigquery_client.py

```diff
@@ -89,58 +163,29 @@ def upload_dataframe(self, dataframe, schema, tablename):
             uri, table_ref, job_config=job_config
         )
 
-        job.result()  # Waits for table load to complete.
+        job.result()
 
-    @refresh_clients
-    def query(self, query, **kwargs):
-        return self.bq_client.query(query, **kwargs)
+    def get_table_column_names(self, project, dataset, table):
+        table_info = self._get_table(project, dataset, table)
+        return [field.name for field in table_info.schema]
 
     @refresh_clients
-    def get_table(self, project, dataset, table):
+    def _get_table(self, project, dataset, table):
         full_table_name = '{}.{}.{}'.format(project, dataset, table)
         return self.bq_client.get_table(full_table_name)
 
-    def get_table_column_names(self, project, dataset, table):
-        table_info = self.get_table(project, dataset, table)
-        return [field.name for field in table_info.schema]
 
-    def download_to_file(self, project, dataset, table, file_path=None, limit=None, offset=None,
-                         fail_if_exists=False, progress_bar=True):
-        if not file_path:
-            file_name = '{}.{}.{}.csv'.format(project, dataset, table)
-            file_path = os.path.join(_USER_CONFIG_DIR, file_name)
+def _rows_to_file(rows, file_path, column_names=None, progress_bar=True):
+    if progress_bar:
+        pb = tqdm.tqdm_notebook(total=rows.total_rows)
```

Comment:
What happens if you run this from a console?

Reply:
No idea. I will check if fails